zerver/models.py

```diff
@@ -2278,6 +2292,7 @@ class Subscription(models.Model):
     # notification settings, stream color, etc., if the user later
     # resubscribes.
     active: bool = models.BooleanField(default=True)
+    is_user_active: bool = models.BooleanField()
```

Comment:
Let's add a comment above this field, noting that it's a denormalization designed to improve the performance of bulk queries of Subscription objects, which usually are only interested in active users.